code-matlab/shared/proc/TuningCurves.m

```diff
@@ -188,25 +191,44 @@
                 spk_hist = conv2(spk_hist,cfg.smoothingKernel,'same');
             end
 
-            spk_hist(tc.no_occ_idx) = NaN;
+            spk_hist(no_occ_idx) = NaN;
             
-            tc.tc2D(iC,:,:) = spk_hist./tc.occ_hist;
+            tc2D(iC,:,:) = spk_hist./occ_hist;
             
-            temp = tc.tc2D(iC,:,:);
-            tc.tc(iC,:) = temp(tc.good_idx);
+            temp = tc2D(iC,:,:);
+            tc(iC,:) = temp(good_idx);
 
         end
 
-        tc.binEdges = cfg.binEdges;
+        binEdges = cfg.binEdges;
         for iDim = 1:nDim
-            tc.binCenters{iDim} = cfg.binEdges{iDim}(1:end-1)+median(diff(cfg.binEdges{iDim}))/2;
-            tc.nBins{iDim} = length(tc.binEdges{iDim})-1;
+            binCenters{iDim} = cfg.binEdges{iDim}(1:end-1)+median(diff(cfg.binEdges{iDim}))/2;
+            nBins{iDim} = length(binEdges{iDim})-1;
         end
 
     otherwise
         error('More than 2 tuning dimensions is not yet implemented.');
 end
 
+%% housekeeping
+tc_out = tc;
+
+% main data
+tc_out.tc = tc;
+if cfg.bootstrap; tc_out.tcboot = tcboot; end
+if exist('tc2D','var'); tc_out.tc2D = tc2D; end
+tc_out.occ_hist = occ_hist;
+tc_out.spk_hist = spk_hist;
+tc_out.binEdges = binEdges;
+tc_out.binCenters = binCenters;
+tc_out.nBins = nBins;
 
+% usr data
+tc_out.usr.good_idx = good_idx;
+tc_out.usr.no_occ_idx = good_idx;
```

Comment:
`good_idx` goes to both of these? doesn't seem right

also, could describe this in help